lms/djangoapps/teams/static/teams/js/views/team_count_detail.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
I agree with Dan that this would probably be better inlined into the one class that uses it.